### PR TITLE
renamathon: rename application to fleet/app

### PR DESCRIPTION
**Pull request by tmigone**
- [x] Update docs
- [x] Update screenshots
- [ ] Update blogposts

Change-type: patch
Signed-off-by: Tomás Migone <tomas@balena.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,12 @@
 
 ## Setup and configuration
 
-Running this project is as simple as deploying it to a balenaCloud application. You can do it in just one click by using the button below:
+Running this app is as simple as deploying it to a balenaCloud application. You can do it in just one click by using the button below:
```
Comment (rahul-thakoor) [inline README.md:13]: Running this app is as simple as deploying it to a balenaCloud ~~application~~ fleet

Comment (tmigone) [reply README.md]: good catch

**Comment by tmigone** [comment]
@balena-ci retest

**Review by maggie0002 — APPROVED** [review]

**Comment by maggie0002** [comment]
@balena-ci retest

**Comment by maggie0002** [comment]
@balena-ci retest

**Pull request merged**